berkeley-mobile/StudyPact/Profile/ProfileViewController.swift

```diff
@@ -164,33 +185,87 @@ extension ProfileViewController {
+        profileView.addGestureRecognizer(tap)
+        
+        profileLabel = UILabel()
+        profileLabel.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        profileLabel.numberOfLines = 0
+        profileLabel.font = Font.bold(30)
+        profileLabel.text = "Profile"
+        profileView.addSubview(profileLabel)
+        profileLabel.translatesAutoresizingMaskIntoConstraints = false
+        profileLabel.topAnchor.constraint(equalTo: profileView.topAnchor, constant: 15).isActive = true
+        profileLabel.leftAnchor.constraint(equalTo: profileView.leftAnchor).isActive = true
+        
+        let button = UIButton()
+        button.setTitle("Logout", for: .normal)
+        button.titleLabel?.font = Font.medium(18)
+        button.setTitleColor(Color.blackText, for: .normal)
+        button.setTitleColor(Color.lightGrayText, for: .selected)
+        
+        profileView.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.centerYAnchor.constraint(equalTo: profileLabel.centerYAnchor).isActive = true
+        button.rightAnchor.constraint(equalTo: profileView.rightAnchor).isActive = true
+        button.addTarget(self, action: #selector(logoutButtonPressed), for: .touchUpInside)
+        
+        button.setHeightConstraint(15)
+        
+        logoutButton = button
+        
+        let radius: CGFloat = 40
         let initials = ProfileLabel(text: "O", radius: radius, fontSize: 40)
         
-        view.addSubview(initials)
+        profileView.addSubview(initials)
         
         initials.translatesAutoresizingMaskIntoConstraints = false
         initials.topAnchor.constraint(equalTo: profileLabel.bottomAnchor, constant: 15).isActive = true
-        initials.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        initials.centerXAnchor.constraint(equalTo: profileView.centerXAnchor).isActive = true
         initials.setHeightConstraint(2 * radius)
         initials.setWidthConstraint(2 * radius)
         
         initialsLabel = initials
         
         let imageView = UIImageView()
-        imageView.layer.cornerRadius = 50
+        imageView.layer.cornerRadius = 40
         imageView.layer.masksToBounds = true
         imageView.clipsToBounds = true
         imageView.isHidden = true
         
-        view.addSubview(imageView)
+        profileView.addSubview(imageView)
 
         imageView.translatesAutoresizingMaskIntoConstraints = false
         imageView.topAnchor.constraint(equalTo: profileLabel.bottomAnchor, constant: 15).isActive = true
-        imageView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
-        imageView.setHeightConstraint(100)
-        imageView.setWidthConstraint(100)
+        imageView.centerXAnchor.constraint(equalTo: profileView.centerXAnchor).isActive = true
+        imageView.setHeightConstraint(80)
+        imageView.setWidthConstraint(80)
```

Comment:
`imageView` is supposed to match the size of `profileView`, I'm assuming. In that case, why not use `radius` and `2 * radius` for the `.cornerRadius` and height/width constraints instead of putting in 40 and 80? Alternatively, for the height and width, you can just match those with the height and width constraints of `initials`.